packages/prisma/schema.prisma

```diff
@@ -110,8 +110,7 @@ model DestinationCalendar {
   externalId   String
   user         User?       @relation(fields: [userId], references: [id])
   userId       Int?        @unique
-  booking      Booking?    @relation(fields: [bookingId], references: [id])
-  bookingId    Int?        @unique
+  booking      Booking[]
```

Comment:
Changed this in destination calendar. Should have been a 1 to many relationship with bookings